api/objects.proto

```diff
@@ -25,6 +26,8 @@ message Meta {
 
 // Node provides the internal node state as seen by the cluster.
 message Node {
+	option (docker.protobuf.plugin.store_object) = { table: "node" };
```

Comment:
Non-blocking dumb question:  I am probably just not seeing it, but where is this option used in the code generation?